src/cmd-prune-containers

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python3 -u
+
+# Prune containers from a remote registry
+# according to the images age
+
+import argparse
+import datetime
+import json
+import re as regexp
+import os
+import subprocess
+from dateutil.relativedelta import relativedelta
+import requests
+import yaml
+
+# Dict of known streams
```

Comment:
Does this have to be a dictionary? from a quit look a list might be enough.

Reply:
yes because i need to associate the stream name with it's id. I tried to do a list of tuples but it was way more complicated.